extensions/amp-sticky-ad/1.0/amp-sticky-ad.css

```diff
@@ -23,6 +23,10 @@ amp-sticky-ad {
   z-index: 11;
   max-height: 100px !important;
   box-sizing: border-box;
+  opacity: 1 !important;
+  background-image: none !important;
+  background-color: #fff;
+  padding-top: 2px;
```

Comment:
Oh, I was experimenting with this PR with the new button style. Failed to clean up. Removed